### ISSUE TITLE
Formating guide. Examples and screen readers

### ISSUE THREAD

**Issue by amazingphilippe** [cds-snc/notification-planning#122]
### Description of issue

To be more accessible, we could add hidden text within the examples to better explain what is going on. Right now, we label the examples to describe the formatting.

> In the following screenshot, the brackets are silent with Voiceover
>
> <img width="691" alt="Capture d’écran, le 2021-10-19 à 11 29 37" src="https://user-images.githubusercontent.com/5230720/137945353-9a008f22-5bb7-4aa2-97df-e08259bf41b4.png">

We should also use hidden text to make sure the special characters are being read properly. 

`Apply now at ((link))`
`Apply now at <span role="presentation">((</span><span class="sr-only"> double opening round brackets </span>link<span class="sr-only"> double closing round brackets </span><span role="presentation">))</span>`

### Potential fix

- What change could fix this issue?
- What nuggets of information could help us fix the issue

### Resources
- Links to documentation or articles that could be helpful
- examples: WCAG docs, ARIA docis, web.dev, Deque, Webaim...

**Comment by amazingphilippe** [comment]
@YedidaZalik I wonder what you think about this one. Basically, we want to make sure that people using screen readers know which characters to input in order to format their templates.

**Comment by YedidaZalik** [comment]
@amazingphilippe is the idea to substitute something else for the following?
 <span role="presentation">((</span><span class="sr-only"> double opening round brackets </span>link<span class="sr-only"> double closing round brackets </span><span role="presentation">))</span>

**Comment by amazingphilippe** [comment]
Yes, the round brackets can be ignored by screen readers. So I think it would be nice to find a way to describe how to format things like links, bolds, italics and headings

**Comment by yaelberger-commits** [comment]
Might want to test with screen reader users to validate if this is a problem, and the best way to describe parentheses in brackets to people who use screen readers
